build/lib/release.sh

```diff
@@ -106,14 +106,14 @@ function kube::release::package_src_tarball() {
     git archive -o "${src_tarball}" HEAD
   else
     local source_files=(
-      $(cd "${KUBE_ROOT}" && find . -mindepth 1 -maxdepth 1 \
+      "$(cd "${KUBE_ROOT}" && find . -mindepth 1 -maxdepth 1 \
         -not \( \
           \( -path ./_\*        -o \
              -path ./.git\*     -o \
              -path ./.config\* -o \
              -path ./.gsutil\*    \
           \) -prune \
-        \))
+        \))"
```

Comment:
original code expects source_files to be an array.
If double quote is added, it becomes single element. So the result is different from original. How about using `while` and `read` according to https://github.com/koalaman/shellcheck/wiki/SC2207?

Reply:
Good point!
Thank you very much.
Changed this according to your suggestion.